### PR TITLE
Normalize use of operators

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11712)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]
@PrestaShop/prestashop-core-developers No need QA. don't see this way to write since maybe 10 years :open_mouth:

File: admin-dev/filemanager/include/utils.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ function foldersize($path)
     $cleanPath = rtrim($path, '/'). '/';
 
     foreach ($files as $t) {
-        if ($t<>"." && $t<>"..") {
+        if ($t!="." && $t!="..") {
```
Comment (jolelievre) [inline admin-dev/filemanager/include/utils.php]: can you please add spaces for readability?
```php
    if ($t != "." && $t != "..")
```

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: admin-dev/filemanager/include/utils.php (hunk 1 above)
Comment (MathiasReker) [reply admin-dev/filemanager/include/utils.php]: Sure

**Review by PierreRambaud — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by jolelievre** [comment]
Thank you @PierreRambaud @MathiasReker

**Pull request merged**